compiler/control/CompileMethod.cpp

```diff
@@ -477,6 +478,6 @@ compileMethodFromDetails(
 
    TR_OptimizationPlan::freeOptimizationPlan(plan);
 
-
+   TR_VerboseLog::vlogRelease();
```

Comment:
We do not need to hold on to the lock up to this point. Other compilation threads will be unnecessarily kept waiting.

Reply:
Where is the acquire for the this release?

Reply:
At [line 277](https://github.com/eclipse/omr/pull/5565/files/a99947fc7810d1ea185da7197a5e37b5f2234324#diff-726d951258a69ea2baa7d8ff8138b4b1R277).